server/public/pluginapi/license.go

```diff
@@ -30,7 +23,7 @@ func isValidSkuShortName(license *model.License) bool {
 	}
 
 	switch license.SkuShortName {
-	case e10, e20, professional, enterprise:
+	case model.LicenseShortSkuE10, model.LicenseShortSkuE20, model.LicenseShortSkuProfessional, model.LicenseShortSkuEnterprise:
```

Comment:
Do we need to expand this to include Premium here also as a valid shortsku and add a new pluginAPI check for minPremium?

Reply:
Done. Thanks for the catch!